test/depots/depotDonneesHomologations.spec.js

```diff
@@ -1679,4 +1682,93 @@ describe('Le dépôt de données des homologations', () => {
         .catch(done);
     });
   });
+
+  describe('sur une demande de nombre moyen de contributeurs', () => {
+    it("peut retourner le nombre moyen de contributeurs pour les services d'un utilisateur donné", (done) => {
+      const referentiel = Referentiel.creeReferentielVide();
+
+      const proprietaire = new ConstructeurUtilisateur().avecIdUtilisateur(
+        'Propriétaire'
+      ).donnees;
+      const premierContributeur =
+        new ConstructeurUtilisateur().avecIdUtilisateur('A').donnees;
+      const secondContributeur =
+        new ConstructeurUtilisateur().avecIdUtilisateur('B').donnees;
+      const troisiemeContributeur =
+        new ConstructeurUtilisateur().avecIdUtilisateur('B').donnees;
+
+      const premierService = new ConstructeurService()
+        .avecId('123')
+        .avecNomService('un premier service').donnees;
+      const secondService = new ConstructeurService()
+        .avecId('456')
+        .avecNomService('un autre service').donnees;
+
+      const adaptateurPersistance =
+        AdaptateurPersistanceMemoire.nouvelAdaptateur({
+          utilisateurs: [
+            proprietaire,
+            premierContributeur,
+            secondContributeur,
+            troisiemeContributeur,
+          ],
+          homologations: [premierService, secondService],
+          autorisations: [
+            new ConstructeurAutorisation()
+              .pourUtilisateur(proprietaire.id)
+              .pourService(premierService.id)
+              .enTantQueCreateur().donnees,
+            new ConstructeurAutorisation()
+              .pourUtilisateur(proprietaire.id)
+              .pourService(secondService.id)
+              .enTantQueCreateur().donnees,
+            new ConstructeurAutorisation()
+              .pourUtilisateur(premierContributeur.id)
+              .pourService(premierService.id)
+              .enTantQueContributeur().donnees,
+            new ConstructeurAutorisation()
+              .pourUtilisateur(secondContributeur.id)
+              .pourService(premierService.id)
+              .enTantQueContributeur().donnees,
+            new ConstructeurAutorisation()
+              .pourUtilisateur(troisiemeContributeur.id)
+              .pourService(premierService.id)
+              .enTantQueContributeur().donnees,
+          ],
+        });
+
+      const depot = DepotDonneesHomologations.creeDepot({
+        adaptateurPersistance,
+        referentiel,
+      });
+
+      depot
+        .nombreMoyenContributeursPourUtilisateur(proprietaire.id)
+        .then((nbMoyenContributeurs) => expect(nbMoyenContributeurs).to.be(1))
```

Comment:
Pourrait-on trouver un jeu de données qui ne donne pas `1` en résultat ?

Reply:
En fait ce test retourne un nombre moyen de `1.333` utilisateurs mais avec Henri, on a vu que l'on voulait l'entier inférieur arrondi le plus proche. Du coup, tu as un expect à `1`.
On pourrait faire en sorte que ce soit supérieur à 1 mais ça demanderait un setup plus important, je ne sais pas si cela te convient

Reply:
Alors à ce moment-là on pourrait dire que le domaine ne fait pas d'arrondi (donc ici on aurait `.to.be(1.5)` me souffle @ThibaudMZN) et passer la responsabilité d'arrondir à l'adaptateur tracking.
Comme ça on perd la précision le plus « loin » possible du domaine.

Reply:
En discutant avec @ThibaudMZN ce matin, et suite à ma remarque hier soir sur Mattermost:
```
Ma première réflexion a été "Oui, c'est vrai ça tient plus de la présentation", jusqu'à ce que cela fasse tilt dans ma tête:
De ce que j'ai compris du besoin d'Henri, la précision décimale ne l'intéresse pas, au niveau métier, c'est une valeur entière qu'il veut.
```
En conclusion, on reste pour l'instant à cette implémentation et is le besoin change un jour, on adaptera